samples/snippets/csharp/tour/arrays/arrays.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <DebugType>portable</DebugType>
```

Comment:
do we need this? I think with csproj all projects are "portable" and you choose your deployment option when running dotnet publish